Assignment4/rectangle.go

```diff
@@ -0,0 +1,51 @@
+/*
+	2. The given program accepts 2 values from the user, length and breadth of a rectangle respectively.
+
+	Complete the program by writing methods "Area" and "Perimeter" on Rectangle type to calculate the area and perimeter of a rectangle.
+
+	Hint:
+	Method Signatures for Area and Perimeter
+	Area() int
+	Perimeter() int
+
+	Formulae:
+	Area = length * width
+	Perimeter = 2 * (length + width)
+
+	Example Test Case 1:
+	Input: 10 20
+	Output:
+	Area of Rectangle: 200
+	Perimeter of Rectangle: 60
+
+*/
+
+package main
+
+import "fmt"
+
+type Rectangle struct {
```

Comment:
If we aren't exporting any variable to another module, then it should be in small-case camel-casing

Reply:
done